### PR TITLE
add content to Learn/Overview - issue #66

### PR DESCRIPTION
**Pull request by veroandreo**
I added content from flyers regarding main features and interfaces to the `General Overview` page ( see #66 ). I believe in this way we highlight some of the things we like most about GRASS GIS. It looks as follows:

image

image

I shrunk all the content into a more readable format by including newlines within paragraphs, as well, that's why the changes seem so massive

### CODE REVIEW

File: content/learn/overview.md [hunk 1]
```diff
@@ -20,36 +26,129 @@ layout: "overview"
...
-<p><b>GRASS GIS</b> capabilities are significantly increased by an impressive list of <b>extensions or add-ons</b>. These modules are contributed by users or developers but are not yet part of the standard distribution.
-  There are currently more than <b>300 extensions</b> in the <a href="https://github.com/OSGeo/grass-addons/tree/master/grass7" >official GRASS Add-ons repo</a> and many others in the wild to perform the most varied type of tasks.
-                    <br>Have a look at the full list of <a href="https://grass.osgeo.org/grass7/manuals/addons/" target="_blank">add-ons manual pages</a> to get an idea. If you don't find what you need, maybe you can <a href="/contribute/development/">develop</a> your own add-on then.
-                    <br>Read <a href="/download/addons/">here</a> to learn how to install them.</p>
+<p><b>GRASS GIS</b> runs on Linux, Mac and Windows. It comes with 
+both an intuitive graphical user interface (GUI) and a command line 
+syntax useful for automation and production.</p>
+<p><b>GRASS GIS</b> contains over 
+<b><a href="https://grass.osgeo.org/grass-devel/manuals/full_index.html" target="_blank">500 modules</a></b>
```
Comment (neteler) [inline content/learn/overview.md:70]: Shall we link to the -devel or -stable manual?

File: content/learn/overview.md [hunk 2]
```diff
@@ -20,36 +26,129 @@ layout: "overview"
...
-                    <br>Read <a href="/download/addons/">here</a> to learn how to install them.</p>
+<p><b>GRASS GIS</b> runs on Linux, Mac and Windows. It comes with 
+both an intuitive graphical user interface (GUI) and a command line 
+syntax useful for automation and production.</p>
+<p><b>GRASS GIS</b> contains over 
+<b><a href="https://grass.osgeo.org/grass-devel/manuals/full_index.html" target="_blank">500 modules</a></b> 
+to process and render geographic data. The software allows to manipulate
+a variety of raster, vector and 3D formats, and run simple to advanced 
+spatial analysis and modeling. GRASS GIS can connect to spatial databases 
+but also interface with a variety of third-party systems and libraries.</p>
```
Comment (neteler) [inline content/learn/overview.md:74]: Mention Jupyter notebooks here?

File: content/learn/overview.md [hunk 3]
```diff
@@ -20,36 +26,129 @@ layout: "overview"
...
+GRASS GIS can be used through different interfaces: 
+<ul>
+<li>The simplest for new users is the <strong>Graphical User Interface</strong> (GUI).</li>
+<li>Power users use the text-based <strong>command line interface</strong> (CLI).</li>
+<li><strong>C API</strong> for programming.</li>
+<li>Scripting library and object-oriented <strong>Python API</strong>.</li>
+<li><strong>Web interface</strong> through WPS servers.</li>
+<li>QGIS provides two different ways to run GRASS GIS modules.</li>
+<li>R provides an interface to GRASS GIS through the package <strong><i>rgrass7</i></strong>.</li>
+</ul>
```
Comment (neteler) [inline content/learn/overview.md:123]: - maybe link to the respective (Wiki) pages?
- add link to https://staging.grass.osgeo.org/learn/tryonline/ ?

File: content/learn/overview.md [hunk 4]
```diff
@@ -20,36 +26,129 @@ layout: "overview"
...
+<br>Have a look at the full list of 
+<a href="https://grass.osgeo.org/grass7/manuals/addons/" target="_blank">add-ons manual pages</a> 
+to get an idea. If you don't find what you need, maybe you can 
+<a href="/contribute/development/">develop</a> your own add-on then.
+Read <a href="/download/addons/">here</a> to learn how to install them.</p>
+</div>
+
+
+<div class="col-lg-6 text-center">
+<img class="bsh" src="https://grass.osgeo.org/grass78/manuals/addons/number_seasons_ndvi.png" width="75%" alt="Output of r.seasons' addon">
```
Comment (neteler) [inline content/learn/overview.md:150]:
```suggestion
<img class="bsh" src="https://grass.osgeo.org/grass-stable/manuals/addons/number_seasons_ndvi.png" width="75%" alt="Output of r.seasons' addon">
```

File: content/learn/overview.md (hunk 3 above)
Comment (veroandreo) [reply content/learn/overview.md:123]: yes, on it :)

File: content/learn/overview.md (hunk 2 above)
Comment (veroandreo) [reply content/learn/overview.md:74]: It's below in the Interfaces section. I can add a `See below` sentence.

File: content/learn/overview.md (hunk 1 above)
Comment (veroandreo) [reply content/learn/overview.md:70]: Oh, I miss that one. I change all to stable now

**Pull request merged**